components/dashboard/src/feedback-form/FeedbackModal.tsx

```diff
@@ -12,13 +12,14 @@ function FeedbackFormModal(props: { onClose: () => void }) {
         props.onClose();
     };
 
-    const onSubmit = () => {
-        props.onClose();
-    };
-
     return (
         <Modal visible={true} onClose={onClose}>
-            <FeedbackComponent onClose={onClose} onSubmit={onSubmit} isModal={true} />
+            <FeedbackComponent
```

Comment:
**issue**: Not sure which change caused this, but the feedback modal now contains different padding and the line separators do not expand towards the edges of the modal as seen on all modals across the product.

| Feedback Modal :x:| New Variable Modal ✔️ |
|-|-|
| <img width="567" alt="Screenshot 2022-05-24 at 12 16 51 PM" src="https://user-images.githubusercontent.com/120486/169997060-5dea415d-bcd0-446c-9bda-7ec02f7923e6.png"> | <img width="560" alt="Screenshot 2022-05-24 at 12 18 59 PM" src="https://user-images.githubusercontent.com/120486/169997386-a7f403db-8398-411a-bfbd-650ab5b2a0af.png"> |